README.md

```diff
@@ -35,8 +38,8 @@ This repo is for our informational website, which will eventually be home to the
 
 To get our static website up and running on your machine, you can follow the commands below:
 
-```
-$ git clone https://github.com/openseattle/cvp
+```bash
```

Comment:
Why are we prefixing everything with `bash` here?

Reply:
Mostly to solve lint errors, basically markdown prefers that you name what to parse the code to.